resources/puppetlabs/lein-ezbake/template/pe/ext/debian/ezbake.init.erb

```diff
@@ -175,18 +183,19 @@ case "$1" in
         get_status_q || exit 0
         do_restart
         ;;
-    restart|force-reload)
-        #
-        # If the "reload" option is implemented then remove the
-        # 'force-reload' alias
-        #
+    restart)
         [ "$VERBOSE" != no ] && log_daemon_msg "Restarting $DESC" "$NAME"
         do_restart
         ;;
+    force-reload|reload)
+        [ "$VERBOSE" != no ] && log_daemon_msg "Reloading $DESC" "$NAME"
+        do_reload
```

Comment:
same comment as in the foss version. we should probably be capturing the return value here to exit with it below.